packages/@aws-cdk/aws-elasticloadbalancingv2/README.md

```diff
@@ -1,2 +1,171 @@
-## The CDK Construct Library for AWS Elastic Load Balancing (V2)
-This module is part of the [AWS Cloud Development Kit](https://github.com/awslabs/aws-cdk) project.
+## AWS Application and Network Load Balancing Construct Library
+
+The `@aws-cdk/aws-elasticloadbalancingv2` package provides constructs for
+configuring application and network load balancers.
+
+### Defining an Application Load Balancer
+
+You define an application load balancer by creating an instance of
+`ApplicationLoadBalancer`, adding a Listener to the load balancer
+and adding Targets to the Listener:
+
+```ts
+import ec2 = require('@aws-cdk/aws-ec2');
+import elbv2 = require('@aws-cdk/aws-elasticloadbalancingv2');
+import autoscaling = require('@aws-cdk/aws-autoscaling');
+
+// ...
+
+const vpc = new ec2.VpcNetwork(...);
+
+// Create the load balancer in a VPC. Set 'internetFacing' to 'false' to
+// create an internal load balancer.
+const lb = new elbv2.ApplicationLoadBalancer(this, 'LB', {
+    vpc,
+    internetFacing: true
+});
+
+// Add a listener and open up the load balancer's security group
+// to the world.
+const listener = lb.addListener('Listener', {
+    port: 80,
+});
+listener.connections.allowDefaultPortFromAnyIpv4('Open to the world');
+
+// Create an AutoScaling group and add it as a load balancing
+// target to the listener.
+const asg = new autoscaling.AutoScalingGroup(...);
+listener.addTargets('Targets', {
```

Comment:
Provide a different ID here in the example, perhaps `addTargets('AppFleet', { port: 8080, targets: [ asg ] })`